frontend/src/Status.jsx

```diff
@@ -81,6 +137,62 @@ class Status extends Component {
             />
           </Panel.Body>
         </Panel>
+        <Panel style={styles.panel}>
```

Comment:
Ideally, this would be a separate component called `SystemRequirementsTable` that we could add to other pages, but since it's only used in one place its fine to leave it as is now.